spec/motion-spec/matchers/respond_to_spec.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::RespondTo" do
+  it 'respond_to passes when subject responds to method name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.